### PR TITLE
Add an Ingest job

### LINKED ISSUE #17: Create ingest job
This should run in sidekiq. It will moves the files from the activestorage to the correct point in the `/dor/assembly` NFS mount, produces the correct XML metadata, and starts the assemblyWF.

Probably something like this: https://github.com/sul-dlss/pre-assembly/blob/master/app/lib/pre_assembly/digital_object.rb#L66-L70

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
Because the processing that needs to be done can take a long time.
Fixes #17

## Was the documentation (README.md, openapi.yml) updated?

yes

### CODE REVIEW

File: app/jobs/ingest_job.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    workflow_client.create_workflow_by_name(druid, 'accessionWF')
+    background_job_result.complete!
+  end
+
+  private
+
+  def workflow_client
+    Dor::Workflow::Client.new(url: Settings.workflow.url,
+                              logger: Rails.logger,
+                              timeout: 60)
```
Comment (mjgiarlo) [inline app/jobs/ingest_job.rb]: Worth extracting to constant or configurable value, or meh?

File: app/jobs/ingest_job.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class IngestJob < ApplicationJob
+  queue_as :default
+
+  def perform(druid:, background_job_result:)
+    background_job_result.processing!
+
+    # TODO: generate contentMetadata.xml
+    # TODO: ship files
+    workflow_client.create_workflow_by_name(druid, 'accessionWF')
+    background_job_result.complete!
```
[the remark this answers is not in the document]
Comment (jcoyne) [reply app/jobs/ingest_job.rb]: Sure.

File: app/jobs/ingest_job.rb (hunk 1 above)
Comment (jcoyne) [reply app/jobs/ingest_job.rb]: very meh

Comment (mjgiarlo) [reply app/jobs/ingest_job.rb]: I buy it.

File: app/jobs/ingest_job.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (mjgiarlo) [reply app/jobs/ingest_job.rb]: thx

**Pull request merged**